### PR TITLE
Simplify build files

### PR DESCRIPTION
**Pull request by jonhawkes**
#build

### CODE REVIEW

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build

File: dev/com.ibm.ws.wsat.migration_fat.3/build.gradle [hunk 1]
```diff
@@ -11,49 +11,18 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 
+apply from: '../com.ibm.ws.wsat.migration_fat.1/common.gradle'
+
 // Define G:A:V coordinates of each dependency
```
Comment (joe-chacko) [inline dev/com.ibm.ws.wsat.migration_fat.3/build.gradle:16]: this comment is no longer relevant

File: dev/com.ibm.ws.wsat.migration_fat.2/build.gradle [hunk 2]
```diff
@@ -11,49 +11,18 @@
...
-    include 'test-applications/**'
-  }
+  requiredLibs project(":com.ibm.ws.wsat.migration_fat.1")
 }
 
-autoFVT.dependsOn project(':com.ibm.ws.wsat.migration_fat.1').autoFVT
+jar.dependsOn copyCommonFiles
+autoFVT.dependsOn copyTestApps
 
 clean.doLast {
```
Comment (joe-chacko) [inline dev/com.ibm.ws.wsat.migration_fat.2/build.gradle:24]: This `clean.doLast` is duplicated in common.gradle. If it works there, it shouldn't be needed here.

File: dev/com.ibm.ws.wsat.migration_fat.2/build.gradle [hunk 3]
```diff
@@ -11,49 +11,18 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 
+apply from: '../com.ibm.ws.wsat.migration_fat.1/common.gradle'
+
 // Define G:A:V coordinates of each dependency
```
Comment (joe-chacko) [inline dev/com.ibm.ws.wsat.migration_fat.2/build.gradle:16]: irrelevant comment

File: dev/com.ibm.ws.wsat.migration_fat.3/build.gradle [hunk 4]
```diff
@@ -11,49 +11,18 @@
...
-    include 'test-applications/**'
-  }
+  requiredLibs project(":com.ibm.ws.wsat.migration_fat.1")
 }
 
-autoFVT.dependsOn project(':com.ibm.ws.wsat.migration_fat.1').autoFVT
+jar.dependsOn copyCommonFiles
+autoFVT.dependsOn copyTestApps
 
 clean.doLast {
```
Comment (joe-chacko) [inline dev/com.ibm.ws.wsat.migration_fat.3/build.gradle:24]: surely not needed in common.gradle _and_ here

File: dev/com.ibm.ws.wsat.migration_fat.4/build.gradle [hunk 5]
```diff
@@ -10,38 +10,19 @@
...
-    }
-  }
-}
 
-addRequiredLibraries.dependsOn addJakartaTransformer
+apply from: '../com.ibm.ws.wsat.migration_fat.1/common.gradle'
 
-jar {
-  dependsOn copyCommonFiles
+// Define G:A:V coordinates of each dependency
```
Comment (joe-chacko) [inline dev/com.ibm.ws.wsat.migration_fat.4/build.gradle:16]: lose this comment

File: dev/com.ibm.ws.wsat.migration_fat.4/build.gradle [hunk 6]
```diff
@@ -10,38 +10,19 @@
...
+  requiredLibs project(":com.ibm.ws.wsat.migration_fat.1")
 }
 
-autoFVT.dependsOn copyCommonFiles
-
-addRequiredLibraries.dependsOn addJakartaTransformer
+jar.dependsOn copyCommonFiles
+autoFVT.dependsOn copyTestApps
 
 clean.doLast {
```
Comment (joe-chacko) [inline dev/com.ibm.ws.wsat.migration_fat.4/build.gradle:24]: duplicated task

File: dev/com.ibm.ws.wsat.migration_fat.5/build.gradle [hunk 7]
```diff
@@ -13,24 +13,22 @@
 
 // Define G:A:V coordinates of each dependency
```
Comment (joe-chacko) [inline dev/com.ibm.ws.wsat.migration_fat.5/build.gradle:14]: lose this comment

File: dev/com.ibm.ws.wsat.migration_fat.5/build.gradle [hunk 8]
```diff
@@ -13,24 +13,22 @@
...
+      into autoFvtDir
+      include 'test-applications/**'
+    }
   }
 }
 
-autoFVT.dependsOn project(':com.ibm.ws.wsat.migration_fat.4').autoFVT
+autoFVT.dependsOn copyTestApps
 
 clean.doLast {
```
Comment (joe-chacko) [inline dev/com.ibm.ws.wsat.migration_fat.5/build.gradle:33]: duplicated task

**Review by joe-chacko — APPROVED** [review]
minor changes suggested — left to developer's discretion

File: dev/com.ibm.ws.wsat.migration_fat.4/build.gradle (hunk 6 above)
Comment (jonhawkes) [reply dev/com.ibm.ws.wsat.migration_fat.4/build.gradle:24]: it's different here

File: dev/com.ibm.ws.wsat.migration_fat.3/build.gradle (hunk 4 above)
Comment (jonhawkes) [reply dev/com.ibm.ws.wsat.migration_fat.3/build.gradle:24]: it's different here

File: dev/com.ibm.ws.wsat.migration_fat.3/build.gradle (hunk 1 above)
Comment (jonhawkes) [reply dev/com.ibm.ws.wsat.migration_fat.3/build.gradle:16]: agreed

File: dev/com.ibm.ws.wsat.migration_fat.2/build.gradle (hunk 2 above)
Comment (jonhawkes) [reply dev/com.ibm.ws.wsat.migration_fat.2/build.gradle:24]: it's different here

File: dev/com.ibm.ws.wsat.migration_fat.2/build.gradle (hunk 3 above)
Comment (jonhawkes) [reply dev/com.ibm.ws.wsat.migration_fat.2/build.gradle:16]: agreed

File: dev/com.ibm.ws.wsat.migration_fat.5/build.gradle (hunk 7 above)
Comment (jonhawkes) [reply dev/com.ibm.ws.wsat.migration_fat.5/build.gradle:14]: agreed

File: dev/com.ibm.ws.wsat.migration_fat.5/build.gradle (hunk 8 above)
Comment (jonhawkes) [reply dev/com.ibm.ws.wsat.migration_fat.5/build.gradle:33]: i don't think so

File: dev/com.ibm.ws.wsat.migration_fat.4/build.gradle (hunk 6 above)
Comment (joe-chacko) [reply dev/com.ibm.ws.wsat.migration_fat.4/build.gradle:24]: fair enough

File: dev/com.ibm.ws.wsat.migration_fat.5/build.gradle (hunk 8 above)
Comment (joe-chacko) [reply dev/com.ibm.ws.wsat.migration_fat.5/build.gradle:33]: ok, so what is different? I see both tasks deleting the `build` dir if present.

Comment (jonhawkes) [reply dev/com.ibm.ws.wsat.migration_fat.5/build.gradle:33]: this one isn't using common.gradle

Comment (joe-chacko) [reply dev/com.ibm.ws.wsat.migration_fat.5/build.gradle:33]: aha

**Pull request merged**